src/lib/Language/Coq/FreeVars.hs

```diff
@@ -14,63 +16,54 @@ module Language.Coq.FreeVars
   , getFreeVars'
   , definedBy
   , NoBinding(..)
-  )
-where
-
-import           Prelude                 hiding ( Num )
+  ) where
 
+import           Prelude hiding ( Num )
 
-import           Data.List.NonEmpty             ( NonEmpty()
-                                                , (<|)
-                                                , partition
-                                                )
-import           Data.List.NonEmpty.Extra       ( appendl )
-import           Data.Set                       ( Set )
-import qualified Data.Set                      as Set
+import           Data.List.NonEmpty ( (<|), NonEmpty(), partition )
+import           Data.List.NonEmpty.Extra ( appendl )
+import           Data.Set ( Set )
+import qualified Data.Set as Set
```

Comment:
The old formatting tool had different columns for the module name and the functions to import. I would prefer it the old way if the new tool has an option for this.
```suggestion
import           Data.List.NonEmpty        ( (<|), NonEmpty(), partition )
import           Data.List.NonEmpty.Extra  ( appendl )
import           Data.Set                  ( Set )
import qualified Data.Set                 as Set
```

Reply:
I tried to make the imports as compact as possible but yes we can enable alignment of import specs.

Reply:
Well, I guess it's a matter of taste which way is better here.